src/OSPSuite.Utility/FileHelper.cs

```diff
@@ -164,15 +165,17 @@ public static string FolderFromFileFullPath(string fileFullPath)
       };
 
       /// <summary>
-      /// Returns the full version of the product defined at the given file full path or null if it did not contain a version information.
-      /// It also returns null if the file does not exists
+      ///    Returns the full version of the product defined at the given file full path or null if it did not contain a version
+      ///    information.
+      ///    It also returns null if the file does not exists
       /// </summary>
       public static Func<string, string> GetVersion = binaryExecutablePath =>
       {
-         if (FileExists(binaryExecutablePath))
-            return FileVersionInfo.GetVersionInfo(binaryExecutablePath).ProductVersion;
+         if (!FileExists(binaryExecutablePath))
+            return null;
 
-         return null;
+         var versionInfo = FileVersionInfo.GetVersionInfo(binaryExecutablePath);
```

Comment:
Also updated this method while I was at it (this is the one that was patched for 7.2.2 in InstallationValidator)